pkg/adapter/pods.go

```diff
@@ -475,6 +475,14 @@ func (r *LocalRuntime) PlayKubeYAML(ctx context.Context, c *cliconfig.KubePlayVa
 	podOptions = append(podOptions, libpod.WithPodName(podName))
 	// TODO for now we just used the default kernel namespaces; we need to add/subtract this from yaml
 
+	if !podYAML.Spec.HostNetwork {
```

Comment:
Podman does not support pod host network, should I remove this `if`?

Reply:
It would support host network, but it would have to be configured for all the networking configuration. Let's leave this check out for this PR, and add an issue on github to track it, so we're not half supporting the option